bears/php/PHPStanBear.py

```diff
@@ -0,0 +1,21 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='phpstan',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)  (?P<message>.*)')
+class PHPStanBear:
+    """
+    Checks the code with ``phpstan analyze``.
+    This can run it on multiple files and folders.
+    See <https://github.com/phpstan/phpstan> for more information.
+    """
+    LANGUAGES = {'PHP'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('analyse', '--no-ansi', filename)
```

Comment:
@satwikkansal well then what can i do with `-c` so the `phpstan` can execute over the file without this error.
By default while running phpstan the level is set to 0.And i had ran that command in cli 
```
phpstan analyse --no-ansi -l 0 /home/damngamerz/bear/hello-world.php
 ------ ----------------------------------------------------------------- 
  Line   hello-world.php                                                  
 ------ ----------------------------------------------------------------- 
  7      Syntax error, unexpected T_ECHO, expecting ';' or '{' on line 7  
 ------ ----------------------------------------------------------------- 

 [ERROR] Found 1 error                                                          
```
Same error reproduced.